shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -657,10 +856,15 @@ void handleTouchExploration(float x, float y) {
         }
     }
 
-    void updateCustomAccessibilityActions(ByteBuffer buffer, String[] strings) {
+    /**
+     * Updates the Android cache of Flutter's currently registered custom accessibility actions.
+     */
+    // TODO(mattcarroll): how are custom actions deleted? this appears to only add them...
```

Comment:
They are currently not deleted. A possible deletion strategy would be to track the semantics node ids that are using an action and delete the action when the last node is removed.

In practice these are used very rarely

Reply:
I added a TODO to consider adding deletion, but I won't worry about that now.  Seems like low priority.